frame/message-queue/Cargo.toml

```diff
@@ -0,0 +1,54 @@
+[package]
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2021"
+name = "pallet-message-queue"
+version = "7.0.0-dev"
+license = "Apache-2.0"
+homepage = "https://substrate.io"
+repository = "https://github.com/paritytech/substrate/"
+description = "FRAME pallet to queue and process messages"
+readme = "README.md"
```

Comment:
Please remove this line since the package doesn't actually have a README.md. That would cause errors during publish. See https://gitlab.parity.io/parity/mirrors/substrate/-/jobs/2139224#L10038.

```
   Uploading pallet-message-queue v7.0.0 (/builds/parity/mirrors/substrate/frame/message-queue)
error: failed to read `readme` file for package `pallet-message-queue v7.0.0 (/builds/parity/mirrors/substrate/frame/message-queue)`
Caused by:
  failed to read `/builds/parity/mirrors/substrate/frame/message-queue/README.md`
```

Reply:
Okay removed.